src/main/webapp/order.xhtml

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:ui="http://java.sun.com/jsf/facelets"
+      xmlns:h="http://java.sun.com/jsf/html"
+      xmlns:p="http://primefaces.org/ui"
+>
+
+<h:head>
+    <title>ToolRent</title>
+</h:head>
+<h:body>
+    <ui:insert name="navbar">
+        <ui:include src="templates/navbar.xhtml"/>
+    </ui:insert>
+
+    <!--https://primefaces.github.io/primefaces/10_0_0/#/components/datepicker
+        https://www.primefaces.org/showcase/ui/input/datepicker/datePicker.xhtml
+        https://stackoverflow.com/questions/9981076/primefaces-how-to-change-the-default-icon-on-the-button-of-the-calendar-field-->
+
+    <script type="text/javascript">
```

Comment:
Maybe put this in resources in folder something like "Locale" to reuse somewhere else later?

Reply:
I moved it to the web-resources folder.